modules/bpi/bpi.images.inc

```diff
@@ -213,3 +229,71 @@ function bpi_fetch_image($filename, $image_url) {
 
   return file_save_data($result->data, $filename, FILE_EXISTS_REPLACE);
 }
+
+/**
+ * Compute the image type from a field name on a node.
```

Comment:
I have gone through the code a couple of times and I still do not understand what the _type_ of an image is. Could you elaborate?